src/adapters/bun.ts

```diff
@@ -21,7 +21,9 @@ class BunServer implements Server<BunFetchHandler> {
   constructor(options: ServerOptions) {
     this.options = options;
 
-    const fetchHandler = wrapFetch(this, this.options.fetch);
+    let fetchHandler = this.options.fetch;
+    fetchHandler = wrapFetchOnError(this.options.fetch, this.options.onError);
+    fetchHandler = wrapFetch(this as unknown as Server, fetchHandler);
```

Comment:
Deno supports `error` as a server option. We don't need to wrap fetch.

Reply:
Addressed in https://github.com/unjs/srvx/pull/48/commits/6353f4c5430581ea5d321f9481ddea741e833f51